### ISSUE TITLE
Add NACK trigger options

### ISSUE THREAD

**Issue by linuxkidd** [danecreekphotography/node-deepstackai-trigger#422]
**Is your feature request related to a problem? Please describe.**
Currently, to use this ai-trigger, one must set up two independent cameras in BI.  This causes a significantly higher resource utilization on the BI box compared to using a single camera with Main and Sub feeds.

**Describe the solution you'd like**
It would be great to have a NACK ( Negative Ack ) URL / MQTT trigger option such that if an image is submitted, but no interested matches found, the NACK trigger will be fired.

In BI, you can call the URL or admin MQTT feed with camera=$cam&flag=0 to cancel an active alert.  Or,  you can use trigger&camera=$cam&flag=1&memo=$summary for a camera trigger event to flag the alert with the match details in the $summary variable.

For more details on this method, see:
https://familytechexec.com/blog/ai-motion-detection-for-blue-iris-new-method-and-free/

**Describe alternatives you've considered**
Currently, only the [VorlonCD/bi-aidetection](https://github.com/VorlonCD/bi-aidetection) version of AITool appears to support the NACK triggers ( called Cancel URL / Cancel Topic+Payload in the 'Actions' config per camera )

**Additional context**
The down-side of using the VorlonCD/bi-aidetection tool is that it does not provide json formatted output, nor does it provide a delayed 'clear' notification.  Thus, it requires extra glue code to make it HomeAssistant friendly.  It also must run on the WIndows box, where I'd prefer only BI to be running and keep all image processing on my much more powerful linux docker environment.

**Comment by danecreekphotography** [comment]
Thanks for the suggestion! Thinking about it for a bit it would be quite a bit of change to add so I'm not likely to get to it anytime soon.

For what it's worth I'm running five 4k cameras on a PC and have no issues with resource utilization. I have substreams and direct to disk recording set up for the HD streams. The SD streams only generate the thumbnails for motion detection.

**Comment by linuxkidd** [comment]
While this would still be a good addition -- I've experienced oddity with how BI handles the flag feature.
- When a flag=1 is received with a memo, it appears to drop all the video bit ahead of when the flag occurs.
- When a flag=0 is received, it of course throws away the whole recording -- but even if it received one or more flag=1 ahead of the flag=0 ( as would be the case with multiple jpgs being flagged by Deepstack, then a final (one or two) which don't show any interested objects.

Ultimately, I've gone back to the original method.  In my initial setup, I had missed adding the sub-stream to the high quality camera.  I don't personally have issue with the resources on my installation, but a buddy did (higher cam count, lower powered BI box).  I've upgraded his BI server and things are running much better :)

FWIW, I did put together a draft of this change, but not being intimate with typescript, it didn't work exactly right.  I can post the code to my fork of your repo if you'd like to see how it could be further revised (to actually work).
